src/lxc/seccomp.c

```diff
@@ -92,6 +92,22 @@ static uint32_t get_v2_default_action(char *line)
 	return ret_action;
 }
 
+static const char *get_action_name(uint32_t action)
+{
+	switch(action & 0xffff0000){
```

Comment:
Usually obvious but it would be good to leave a short comment stating that you're trying to check whether any of the upper 16 bits are high.